setup/mysql.sql

```diff
@@ -198,12 +205,18 @@ ALTER TABLE `domains` ENABLE KEYS;
 UNLOCK TABLES;
 
 --
+-- Generate password for siteadmin
+--
+
+SET @SITE_ADMIN_PW = SUBSTRING(TO_BASE64(LOWER(CONV(FLOOR(RAND() * 99999999999999), 10, 36))),1,10);
+
+--
 -- Seed the `users` table with the siteadmin user
 --
 
 LOCK TABLES `users` WRITE;
 ALTER TABLE `users` DISABLE KEYS;
-INSERT INTO `users` VALUES (1,1,'siteadmin','siteadmin','$6$uR.EiB1dj5rrvwMF$Qh5LgdjOZavKXwhi9IF0Yuzu7qxsG.dLTTB8e./55ZRNfBuZVLnfUSOEXa0oWT6174myO.WYkOy83HYWAKPbK/',65535,65535,'','','site',1,0,0,0,0,0,0,0,1,NULL,NULL,0,0,0,'SiteAdmin',0,0,NULL,NULL);
+INSERT INTO `users` VALUES (1,1,'siteadmin','siteadmin',ENCRYPT(@SITE_ADMIN_PW,CONCAT(@PW_PREFIX,MD5(RAND()))),65535,65535,'','','site',1,0,0,0,0,0,0,0,1,NULL,NULL,0,0,0,'SiteAdmin',0,0,NULL,NULL);
```

Comment:
> There is a risk that the user will keep this password because it "looks" secure.

OK, let's leave the prefix as-is. After all, *most* systems support SHA-512.

>On the other side, a sha512-pw with a longer hash does require only some work on the mysql-creation-scripts, so less testing.

Actually, I just checked and the salt is long enough, or at least it seems to be in most cases (above 40 characters, whereas even `bcrypt` needs just 22, and `SHA-512` [needs 16](http://php.net/crypt)).

I've just updated the PR to have mixed-case salts. I think this is good to go.

> For vexim3, I'd like to see some kind of installer-script in php. So it creates the database for you.

Me too.